text/0000-no-more-extern-crate.md

```diff
@@ -70,29 +85,21 @@ For example, `cargo build`-ing a crate `my_crate` with a dependency on `rand`
 results in a call to rustc that looks something like
 `rustc --crate-name mycrate src/main.rs --extern rand=/path/to/librand.rlib ...`.
 
-When an external crate is specified this way,
-the crate will automatically brought into scope as if an
-`extern crate name_of_crate;`
-declaration had been added to the current crate root.
-This behavior won't occur when including a library using the `-l`
+When an external crate is specified this way, it will be automatically
+available to any module in the current crate through `use` statements or
+absolute paths (e.g. `::rand::random()`). It will _not_ be automatically
+imported at root level as happens with current `extern crate`.
```

Comment:
I understand the end goal of the updated proposal - make the access to implicitly injected (extern crate) names uniform across all modules, including the root one.
And this is a reasonable goal, this is what the prelude does right now.
I'll think how to do it while keeping the formalist in me happy.
I still think the rules similar to described in the RFC will cause WTF reactions in people learning the language more often than existing "local/global non-use/use" distinction.